### PR TITLE
Format citations and correct command template

### PR DESCRIPTION
**Pull request by LilyAnderssonLee**
1. Correct minor format issues based on release reviews. 
2. Correct command template in **main.nf**
3. Add information to CHANGELOG.md about adding autoMounts to apptainer profile in **nextflow.config**

## PR checklist

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/taxprofiler/tree/master/.github/CONTRIBUTING.md)- [ ] If necessary, also make a PR on the nf-core/taxprofiler _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [ ] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: main.nf [hunk 1]
```diff
@@ -31,7 +31,7 @@ include { validateParameters; paramsHelp } from 'plugin/nf-validation'
 if (params.help) {
     def logo = NfcoreTemplate.logo(workflow, params.monochrome_logs)
     def citation = '\n' + WorkflowMain.citation(workflow) + '\n'
-    def String command = "nextflow run ${workflow.manifest.name} --input samplesheet.csv --genome GRCh37 -profile docker"
+    def String command = "nextflow run ${workflow.manifest.name} --input samplesheet.csv --databases databases.csv --outdir <OUTDIR> -profile docker --run_kraken2"
```
Comment (sofstam) [inline main.nf:34]: Shall we have a name for the output directory?

Comment (LilyAnderssonLee) [reply main.nf:34]: We could do that actually!

Comment (LilyAnderssonLee) [reply main.nf]: Done!

File: main.nf [hunk 2]
```diff
@@ -31,7 +31,7 @@ include { validateParameters; paramsHelp } from 'plugin/nf-validation'
 if (params.help) {
     def logo = NfcoreTemplate.logo(workflow, params.monochrome_logs)
     def citation = '\n' + WorkflowMain.citation(workflow) + '\n'
-    def String command = "nextflow run ${workflow.manifest.name} --input samplesheet.csv --genome GRCh37 -profile docker"
+    def String command = "nextflow run ${workflow.manifest.name} --input samplesheet.csv --databases databases.csv --outdir results -profile docker --run_kraken2"
```
Comment (jfy133) [inline main.nf:34]:
```suggestion
    def String command = "nextflow run ${workflow.manifest.name} --input samplesheet.csv --databases databases.csv --outdir results/ -profile docker --run_kraken2"
```

**Review by jfy133 — APPROVED** [review]

**Pull request merged**